wallet/mixing.go

```diff
@@ -46,7 +46,10 @@ func init() {
 	}
 }
 
-var errNoSplitDenomination = errors.New("no suitable split denomination")
+var (
+	errNoSplitDenomination = errors.New("no suitable split denomination")
+	errNoSemaphoresAvail   = errors.New("no semaphores available for denomination")
```

Comment:
instead of "no semaphores available" (which isn't entirely accurate or makes sense), how about "semaphore count for denomination reached", or even just "throttled mix request for split denomination".